android/src/main/AndroidManifest.xml

```diff
@@ -13,6 +13,13 @@
         android:version="1"
         tools:node="replace"/>
 
+    <!--
+    Passthrough feature flag.
+    `required` should be `true` when passthrough is required for app functionality, `false` otherwise.
+    -->
+    <uses-feature android:name="com.oculus.feature.PASSTHROUGH"
+        android:required="false" />
```

Comment:
Does this mean this feature only works when use `AAR` or can we add this to Godot itself?

Reply:
We can this to Godot itself.

Most of the Godot mobile VR options have been for the VrApi plugin, so we need to figure out how to introduce configuration options for the OpenXR plugin in a way that doesn't confuse users (e.g: Do we deprecate the VrApi options in the next 3.x release?).

In addition, since the OpenXR plugin is multi-platform, we need to figure out a way to inform the user when a selected configuration is not available on the target platform.

Reply:
I think we need to keep supporting VrAPI until people have switched over but indeed, we need to figure out a nicer way to switch the different options in/out of the config.

As for what is and isn't supported in OpenXR itself, that one is harder especially for people who are making cross platform games. What is available on the machine they are working on, and what is available on the different target platforms they are deploying on differs.

In my eyes this is a combo between making sure features that aren't supported have good default behavior and exposing what is supported through the configuration object. Right now it already has the supported features string that we build as we enable extensions, but I think some of those need to be simplified to simple booleans we can query. So a "supports_handtracking" bool and a "supports_passthrough" bool, so the person building a game can react accordingly.